independent-projects/arc/processor/src/test/java/io/quarkus/arc/processor/DotNamesTest.java

```diff
@@ -1,20 +1,22 @@
 package io.quarkus.arc.processor;
 
 import static io.quarkus.arc.processor.Basics.index;
-import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 import io.quarkus.arc.processor.DotNamesTest.Nested.NestedNested;
 import java.io.IOException;
 import org.jboss.jandex.DotName;
 import org.jboss.jandex.Index;
-import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
 
 public class DotNamesTest {
 
     @Test
     public void testSimpleName() throws IOException {
         Index index = index(Nested.class, NestedNested.class, DotNamesTest.class);
-        assertEquals("Nested", DotNames.simpleName(index.getClassByName(DotName.createSimple(Nested.class.getName()))));
+        Assertions.assertEquals("Nested",
```

Comment:
Could we use static imports everywhere?

Reply:
Some tests used them, some didn't. I merely converted the imports into Junit5 style.
Is there any benefit to doing this? I mean, next time I write a test, I am pretty sure I will write it without static imports yet again ;)

Reply:
Aesthetics and uniformity ;-). Ok, just ignore my comment...